packages/core/src/App/Containers/UpdateEmailModal/update-email-modal.jsx

```diff
@@ -82,7 +71,13 @@ const UpdateEmailModal = ({
                         />
                     </Text>
                     <Modal.Footer className='change-email-update__footer'>
-                        <Button onClick={onClickButton} has_effect text={localize('Log in')} primary large />
+                        <Button
+                            onClick={() => redirectToLogin(false, getLanguage())}
```

Comment:
@shontzu-deriv, don't we need to add the `false` in  `redirectToLogin` here as well? 🤔 
```suggestion
                            onClick={() => redirectToLogin(false, getLanguage(), false)}
```

Reply:
acked !